### DISCUSSION TITLE
lineare algebra

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#73] [General]
Bestimmung einer Basis von Ker A

Schritt 1: Verwende den Gauss ALgorithmus um A auf Zeilenstufenform R zu bringen A Gauss R

- Schritt 2: Dies definiert 

- Schritt 3: 

Bsp: A = ()
- [ ] Wie würden Sie dieses Problem angehen

mit x_2 = alpha 
x_5 = betha 
erhalten wir x_4 = 2_btha 

2_x_3 = 
= alpha + 11 

was gibt mit den 
2 -1 3 -1 1 
2 -1 3 0 -1 
-4 2 -4 5 -5 
0 0 -2 2 -7 

-> 

(2 -1 3 -1 1)
(0 0 2 3 -3)
(0 0 0 1 -2)
()

Rang einer Matrix A ist in R^m*n
R - Zeilenstufenform 
rangA = # Pivots in Zeilenstufenform R
= dim ImA = dim des spaltenfaums = dim(span{Zeilen von A}).

2) Zeilenraum von A = Spaltenraum von A^T

Rang(A^T) = Im

- Kap1: 
5.2: Rang A = dim(Im A)
- kap2: 
- kap3

Schnitt 1: Verwende das gauss alg um A auf Zeilenstufenform R zu bringen. 

Schnitt 2: Diese definiert die Pivotspalten. 

Schnitt 3: Vergiss R, und betrachte von A die Pivotspalten, die denn sind eine basis von BildvonA bilden.

Bsp: 

A = (1 2 3 4 )
(1 7 9 11)
(2 -1 0 1)

**Comment by esthicodes** [comment]
Def.
Eine Räume und die allgemeine Lösungen eines LGS A_x = b.

Def.
Sei V ein V*R üner K
U ein Unterraum von V
und v_0 ist in V
die Menge v_0 + U := {v_0+ u | u ist in U }

**Reply by esthicodes** [reply]
![image](https://user-images.githubusercontent.com/78131082/200805874-2fb42409-9a77-4965-8d6e-0b5c367e265d.jpeg)

**Comment by lkxiluo** [comment]
您好！邮件我已收到。
